src/manager/event_handlers.ts

```diff
@@ -442,6 +449,6 @@ function refreshRoundedCorners(actor: ExtensionsWindowActor): void {
 
 /** Refresh rounded corners settings for all windows. */
 function refreshAllRoundedCorners() {
-    global.get_window_actors().forEach(refreshRoundedCorners);
+    global.get_window_actors().forEach(actor => refreshRoundedCorners(actor));
```

Comment:
I already fixed this in https://github.com/flexagoon/rounded-window-corners/commit/98096c7b844950478469581114f3f45416718219, you should rebase on top of the latest commit